targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F469xI/TARGET_SDP-K1/PinNames.h

```diff
@@ -0,0 +1,429 @@
+    D11         = PA_7,
+    D12         = PB_4,
+    D13         = PB_3,
+    D14         = PB_7,// added for the I2C SDA line
+    D15         = PB_8,
+
+    // STDIO for console print
+#ifdef MBED_CONF_TARGET_STDIO_UART_TX
+    STDIO_UART_TX = MBED_CONF_TARGET_STDIO_UART_TX,
+#else
+    STDIO_UART_TX = PC_12,
+#endif
+#ifdef MBED_CONF_TARGET_STDIO_UART_RX
+    STDIO_UART_RX = MBED_CONF_TARGET_STDIO_UART_RX,
+#else
+    STDIO_UART_RX = PD_2,
+#endif
+
+	// Debug pins
+	DEBUG_GPIO0	= PG_6,
+    // Generic signals namings
+    LED1        = PK_4,	// Status LED
+    LED2        = PK_7, // Red LED
+    LED3        = PK_6, // Orange LED
+    LED4        = PK_5, // Green LED
+    AWAKE        = PK_3,	
+	//Removing hardware unsupported features
+    //LED_RED     = LED1,
+    //USER_BUTTON = PA_0,
+    // Standardized button names
+    //BUTTON1 = USER_BUTTON,
+    SERIAL_TX   = STDIO_UART_TX,
+    SERIAL_RX   = STDIO_UART_RX,
+    USBTX       = STDIO_UART_TX,
+    USBRX       = STDIO_UART_RX,
+    I2C_SCL     = PB_8,
+    I2C_SDA     = PB_7,
+    SPI_MOSI    = PA_7,
+    SPI_MISO    = PB_4,
+    SPI_SCK     = PB_3,
+    SPI_CS      = PA_15,	// SPI pins to Arduino connector
+    PWM_OUT     = PA_1,
+
+    /**** USB pins ****/
+    /*USB_OTG_FS_DM = PA_11,
+    USB_OTG_FS_DP = PA_12,
+    USB_OTG_FS_ID = PA_10,
+    USB_OTG_FS_SOF = PA_8,
+    USB_OTG_FS_VBUS = PA_9,*/
+    USB_OTG_HS_DM = PB_14,
+    USB_OTG_HS_DP = PB_15,
+    USB_OTG_HS_ID = PB_12,
+    USB_OTG_HS_SOF = PA_4,
+    USB_OTG_HS_ULPI_CK = PA_5,
+    USB_OTG_HS_ULPI_D0 = PA_3,
+    USB_OTG_HS_ULPI_D1 = PB_0,
+    USB_OTG_HS_ULPI_D2 = PB_1,
+    USB_OTG_HS_ULPI_D3 = PB_10,
+    USB_OTG_HS_ULPI_D4 = PB_11,
+    USB_OTG_HS_ULPI_D5 = PB_12,
+    USB_OTG_HS_ULPI_D6 = PB_13,
+    USB_OTG_HS_ULPI_D7 = PB_5,
+    USB_OTG_HS_ULPI_DIR = PI_11,
+    USB_OTG_HS_ULPI_DIR_ALT0 = PC_2,
+    USB_OTG_HS_ULPI_NXT = PH_4,
+    USB_OTG_HS_ULPI_NXT_ALT0 = PC_3,
+    USB_OTG_HS_ULPI_STP = PC_0,
+    USB_OTG_HS_VBUS = PB_13,
+
+    /**** ETHERNET pins ****/
+    /*ETH_COL = PH_3,
+    ETH_COL_ALT0 = PA_3,
+    ETH_CRS = PH_2,
+    ETH_CRS_ALT0 = PA_0,
+    ETH_CRS_DV = PA_7,
+    ETH_MDC = PC_1,
+    ETH_MDIO = PA_2,
+    ETH_PPS_OUT = PB_5,
+    ETH_PPS_OUT_ALT0 = PG_8,
+    ETH_REF_CLK = PA_1,
+    ETH_RXD0 = PC_4,
+    ETH_RXD1 = PC_5,
+    ETH_RXD2 = PH_6,
+    ETH_RXD2_ALT0 = PB_0,
+    ETH_RXD3 = PH_7,
+    ETH_RXD3_ALT0 = PB_1,
+    ETH_RX_CLK = PA_1,
+    ETH_RX_DV = PA_7,
+    ETH_RX_ER = PI_10,
+    ETH_RX_ER_ALT0 = PB_10,
+    ETH_TXD0 = PG_13,
+    ETH_TXD0_ALT0 = PB_12,
+    ETH_TXD1 = PG_14,
+    ETH_TXD1_ALT0 = PB_13,
+    ETH_TXD2 = PC_2,
+    ETH_TXD3 = PE_2,
+    ETH_TXD3_ALT0 = PB_8,
+    ETH_TX_CLK = PC_3,
+    ETH_TX_EN = PG_11,
+    ETH_TX_EN_ALT0 = PB_11,
+*/
+    /**** OSCILLATOR pins ****/
+    RCC_OSC32_IN = PC_14,
+    RCC_OSC32_OUT = PC_15,
+  /*  RCC_OSC_IN = PH_0,
+    RCC_OSC_OUT = PH_1,
+*/
+    /**** DEBUG pins ****/
+    SYS_JTCK_SWCLK = PA_14,
+  /*  SYS_JTDI = PA_15,
```

Comment:
why are some pinc commented out? If they are not valid, remove. Otherwise a comment in the header would help the intention leaving them there

Reply:
https://github.com/malavikasajikumar/mbed-os/commit/5e9e1404c8007202e3d2392bea906774932c37c6